src/runtime/hexagon_remote/Makefile

```diff
@@ -78,6 +79,10 @@ remotes: bin/v60/libhalide_hexagon_remote_skel.so bin/v60/hexagon_sim_remote
 bin/src/halide_hexagon_remote.h bin/src/halide_hexagon_remote_skel.c bin/src/halide_hexagon_remote_stub.c: halide_hexagon_remote.idl
 	mkdir -p $(@D)
 	$(HEXAGON_QAIC) $(HEXAGON_QAICFLAGS) $^ -o $(@D)
+	mv -f bin/src/halide_hexagon_remote.h      bin/src/halide_hexagon_remote.h-orig 
```

Comment:
I agree.  Sorry about the Makefile mess, it was the only way I could find to deal with the uint64/int64 types that qaic was generating.  I like the idea of changing the param to bwMegabytesPerSec.  I'll go ahead and make the change.